synapse/lib/storm.lark

```diff
@@ -315,6 +315,7 @@ _condexpr: _cond | orexpr | andexpr
 orexpr: _condexpr _WS "or" _WS _cond
 andexpr: _condexpr _WS "and" _WS _cond
 
+TRIPLEQUOTEDSTRING: /'''.*?'''/s
```

Comment:
This doesn't allow a triple quoted string to end with an escaped single quote:
```
cli> storm $x='''foo=\'bar\''''
Executing query at 2022/01/21 17:29:17.057
$x='''foo=\'bar\''''
                   ^
Syntax Error: No terminal defined for ''' at line 1 col 20.  Expecting one of: whitespace or comment, {, |

Reply:
I think
```
/'''.*(?:\\')'''/s
```
would work

Reply:
any suggestions? :D